geode-docs/reference/topics/gemfire_properties.html.md.erb

```diff
@@ -489,7 +489,7 @@ off-heap-memory-size=120g</code></pre></td>
 <tr>
 <td>redis-bind-address</td>
 <td>Specifies the address on which the Redis API for <%=vars.product_name%> is listening. If set to the empty string or this property is not
-specified, localhost is requested from the operating system.</td>
+specified, the server will listen on all local addresses.
```

Comment:
For consistency and ease of understanding, I recommend present tense:

...the server listens on all local addresses.

I'd like to see this change adopted for the doc (.erb) file, at least. I recommend it for the .java files, as well.

Reply:
Just updated the doc and Java docs!  Thank you, Dave.